proposals/lip-0060.md

```diff
@@ -106,6 +106,8 @@ The block assets property is validated as follows:
   * Check that each entry in the assets array has `module` corresponding to a module registered in the chain.
   * The individual entries are checked by the respective modules.
 
+Note that in contrast to the block processing in [LIP 0055][lip-0055#asset-validation], for the genesis block there is no limitation on the size of the data property of every entry.
```

Comment:
```diff
Note that in contrast to the block processing in [LIP 0055][lip-0055#asset-validation], for the genesis block there is no limitation on the size of the data property of every entry.
```

Reply:
Furthermore, I would add the sentence to the first bullet point of the static validation (since the difference is actually there)

Reply:
done